### PR TITLE
Windows Merge PR

### PR DESCRIPTION
**Pull request by Thuan2000**
This PR will merge the forked repo's windows branch into Bassa's windows branch. 

It contains modified sourcecode compatible with windows as well as the README, which contains specific instructions on how to set up and use Bassa on Windows. 

- Thuan Nguyen

### CODE REVIEW

**Comment by malithsen** [comment]
@Thuan2000 README.md now conflicts with the existing. Can you resolve them as well?

File: components/core/UserManager.py [hunk 1]
```diff
@@ -38,9 +37,9 @@ def check_user_name(username):
 def get_user(username):
     db = threadpool.connect()
     if db is not None:
-        cursor = db.cursor(MySQLdb.cursors.DictCursor)
+        cursor = db.cursor()#MySQLdb.cursors.DictCursor)
         sql = "SELECT * FROM user WHERE user_name='%s' and blocked=0;" % username
-        cursor.execute(sql)
+        cursor.execute(sql)
```
Comment (agentmilindu) [inline components/core/UserManager.py]: If you remove the tailing whitespace here, Codacy linting will pass. 

https://www.codacy.com/app/agentmilindu/Bassa/pullRequest?prid=476397#

**Comment by agentmilindu** [comment]
Travis build  keeps failing for this PR. Can someone check why? Something wrong with the PhanthomJS tests.
https://travis-ci.org/scorelab/Bassa/builds/191204480

**Comment by agentmilindu** [comment]
@rehrumesh @malithsen Can we merge this one?

**Pull request merged**